spec/factories/dfe_sign_in/users.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :dfe_signin_user, class: "DfeSignIn::User" do
+    dfe_sign_in_id { "7516e8f7-ab90-4eb6-a998-b798e835ca3d" }
```

Comment:
This should be generated rather than a fixed string, otherwise all the factory-created users will have the same unique identifier.